jupyterlab/run.sh

```diff
@@ -15,11 +19,17 @@ if ! command -v jupyterlab > /dev/null 2>&1; then
   fi
   # install jupyterlab
   pipx install -q jupyterlab
-  echo "🥳 jupyterlab has been installed\n\n"
+  printf "%s\n\n" "🥳 jupyterlab has been installed"
 else
-  echo "🥳 jupyterlab is already installed\n\n"
+  printf "%s\n\n" "🥳 jupyterlab is already installed"
 fi
 
-echo "👷 Starting jupyterlab in background..."
-echo "check logs at ${LOG_PATH}"
-$HOME/.local/bin/jupyter-lab --ServerApp.ip='0.0.0.0' --ServerApp.port=${PORT} --no-browser --ServerApp.token='' --ServerApp.password='' > ${LOG_PATH} 2>&1 &
+printf "👷 Starting jupyterlab in background..."
+printf "check logs at ${LOG_PATH}"
+$HOME/.local/bin/jupyter-lab --no-browser \
+  "$BASE_URL" \
```

Comment:
Very minor nit, could we rename this to `BASE_URL_FLAG` or something?  I was confused at first because I thought we were putting the base URL directly in the command line invocation.  :laughing:

Reply:
Or we use `--ServerApp.base_url=${BASE_URL}` directly here.  I think a blank string is valid.

Reply:
Usually the default value is `/` and not an empty string for some other tools I have seen. Like filebrowser and MS code-server.

Reply:
`/` makes sense to me!